### PR TITLE
Updating `lib/core/README.md` about Dispatching requests

### PR DESCRIPTION
**Pull request by wafuwafu13**
Hi :) 
I read code, and supplemented the document.

### CODE REVIEW

File: lib/core/README.md [hunk 1]
```diff
@@ -3,5 +3,6 @@
 The modules found in `core/` should be modules that are specific to the domain logic of axios. These modules would most likely not make sense to be consumed outside of the axios module, as their logic is too specific. Some examples of core modules are:
 
 - Dispatching requests
+  - requests send at `adapters/` (see lib/adapters/README.md).
```
Comment (jasonsaayman) [inline lib/core/README.md:6]: Can you change this to read:

> Requests sent via `adapters/` (see lib/adapters/README.md)

**Review by jasonsaayman — COMMENTED** [review]
Please check out the comment I left.

File: lib/core/README.md (hunk 1 above)
Comment (wafuwafu13) [reply lib/core/README.md]: I fixed :) https://github.com/axios/axios/pull/3772/commits/9e5245668a7a1074d88d57c3828c3ad7eb057b74

**Review by jasonsaayman — APPROVED** [review]
👍

**Pull request merged**